components/nodemanager-service/api/nodes/nodes.proto

```diff
@@ -110,6 +110,15 @@ message Node {
 	string state = 26;
 	string name_prefix = 27;
 	repeated string projects = 28;
+	LastContactData run_data = 29;
+	LastContactData scan_data = 30;
+}
+
+message LastContactData {
+	string id = 1; // run id or report id
+	string status = 2;
+	string penultimate_status = 3;
```

Comment:
If these are always one of a set of values, should it be an enum or similar?

Reply:
yes, good point -- status and penultimate status should be an enum of passed, failed, skipped.  and probably empty string as the default